contracts/EthlanceIssuer.sol

```diff
@@ -1,22 +1,27 @@
 pragma solidity ^0.5.0;
 
 import "./StandardBounties.sol";
+import "./EthlanceJobs.sol";
 import "./token/IERC20.sol";
 import "./token/IERC721.sol";
 
-contract EthlanceBountyIssuer {
+contract EthlanceIssuer {
 
   StandardBounties internal constant standardBounties = StandardBounties(0xfEEDFEEDfeEDFEedFEEdFEEDFeEdfEEdFeEdFEEd);
+  EthlanceJobs internal constant ethlanceJobs = EthlanceJobs(0xfEEDFEEDfeEDFEedFEEdFEEDFeEdfEEdFeEdFEEd);
```

Comment:
use a different placeholder - when deploying it is replaced by exact match in the bytecode, so no way to distinguish between the two!

Reply:
good catch